english/interact/interact.md

```diff
@@ -0,0 +1,16 @@
+You turn toward the marshmallow hand.
+
+The marshmallow hand retreats towards the other end of the room.
+
+You take a step forward to see how the marshmallow hand will react...
+
+The marshmallow hand sinks further into the wall behind him.
+
+You close your eyes and contemplate your next step.
+
+When you open your eyes, the marshmallow hand is nowhere to be seen....
+
+You then:
+
+[Take your lighter out] (lighter/lighter.md)
+[Bend down on one knee] (bend/bend.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.